SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -549,12 +563,73 @@ FunctionDef {
 		^().putPairs(this.keyValuePairsFromArgs)
 	}
 
+
+	makePerformableArray {|argumentsArray=([]), keywordArgumentEnvir=(()), variableArgumentsArray=([])|
```

Comment:
I've put it here in FunctionDef because otherwise you have to duplicate it about in both object and function. Since it is somewhat fiddly, and I might try and turn it into a primitive, it is better to have it in one place.